Bottom: 1efd1a63fce0a680ae423da3f98f3397659c6870
Top:    b9081271368fc3fd8f9a8c571292c9a6b4e2ea1d
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-03 15:03:21 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/direct.c b/daemon/direct.c
index b52f52c..1f5266c 100644
--- a/daemon/direct.c
+++ b/daemon/direct.c
@@ -38,8 +38,6 @@
 #define INCLUDE_PENDING_FUNCTIONS
 #include "automount.h"
 
-extern unsigned long mountflags;
-
 /* Attribute to create detached thread */
 extern pthread_attr_t th_attr_detached;
 
diff --git a/daemon/indirect.c b/daemon/indirect.c
index 131f6f0..02a1de6 100644
--- a/daemon/indirect.c
+++ b/daemon/indirect.c
@@ -37,8 +37,6 @@
 #define INCLUDE_PENDING_FUNCTIONS
 #include "automount.h"
 
-extern unsigned long mountflags;
-
 /* Attribute to create detached thread */
 extern pthread_attr_t th_attr_detached;
